### PR TITLE
Update docu with uint16_t as void *

### PR DESCRIPTION
**Pull request by markterrill**
This trips up experienced programmers let alone beginners!

### CODE REVIEW

File: mongoose-os/api/core/mgos_timers.h.md [hunk 1]
```diff
@@ -76,6 +76,19 @@ mgos_timer_id mgos_set_timer(int msecs, int flags, timer_callback cb,
 >   return MGOS_APP_INIT_SUCCESS;
 > }
 > ```
+> Example passing unsigned int as an argument to void *
+> ```c
+> static void my_timer_cb(void *arg) {
+>   uint16_t conn_id = (intptr_t) arg;
```
Comment (rojer) [inline mongoose-os/api/core/mgos_timers.h.md]: uintptr_t

Comment (markterrill) [reply mongoose-os/api/core/mgos_timers.h.md]: Sorry, I don't understand.

Comment (markterrill) [reply mongoose-os/api/core/mgos_timers.h.md]: oh! cool, spotted.

**Comment by rojer** [comment]
merged, thanks!

**Pull request merged**